### ISSUE TITLE
Lazier

### ISSUE THREAD

**Issue by maelle** [LucyMcGowan/lazyacademic#1]
Such a cool idea! Haven't tried it yet but am considering it for https://github.com/dacornu/website (not mine so maybe I won't )

I was wondering whether the package could also populate a .tex/md for the publications section of a resume?

And I day-dreamed about using `tic` to have the code run on Travis (at each push, and triggered by hand /with a hook when the sheet is updated (Zapier?) So you'd update the website and the pdf cv there at once.

In the end it'd cost more time setting it up though 😂 but I might read and use your code!

**Comment by LucyMcGowan** [comment]
YES -- I am actually working on that locally (populating a resume)! It is still very much a WIP, but will push as soon as I have something going! I don't know enough at the moment about getting it to run on travis, but would love help if you'd be interested!

**Comment by maelle** [comment]
I might try experimenting as time allows 😀

I have used Travis deploys only for https://github.com/ropensci/dev_guide#technical-details but know of examples involving Netlify.

**Comment by maelle** [comment]
I guess one could ask Travis to push to a website branch and tell Netlify to build from that instead of from master .

Or let Travis push to master like https://github.com/krlmlr/tic.blogdown/commits/master

**Comment by maelle** [comment]
See also https://github.com/ropenscilabs/travis#travis
